src/engine/Engine.ts

```diff
@@ -47,6 +47,24 @@ export enum DisplayMode {
    Position
 }
 
+/**
+ * Enum representing the different mousewheel event bubble prevention
+ */
+export enum PageScrollPreventionMode {
```

Comment:
Let's omit the word`Page` from `PageScrollPreventionMode` to make it simpler. `ScrollPreventionMode` is pretty clear